docs/guides/define_agent.md

```diff
@@ -200,3 +200,53 @@ If you have [set up the local registry](./set_up.md#set-up-the-local-registry) w
     ```bash
     autonomy packages lock
     ```
+
+4. **(Optional) Run and test the agent locally.** We will show you how to run a service deployment with multiple agents in the next guide. However, you can also run a standalone version (that is, without being part of a service) of the agent you have created. This can be useful to test and debug certain features quickly.
+
+      1. In the workspace folder, fetch the agent from the local registry:
+
+         ```bash
+         autonomy fetch your_name/your_agent:0.1.0 --local
+         ```
+
+      2. Provision your agent with an Ethereum private key:
+
+         ```bash
+         cd your_agent
+         echo -n "0x47e179ec197488593b187f80a00eb0da91f1b9d0b13f8733639f19c30a34926a" > your_agent_key.txt
+         autonomy add-key ethereum your_agent_key.txt
+         ```
+
+         If your agent is using an {{fsm_app}}, you also need to override the variable `all_participants` in the `aea-config.yaml` file with the wallet address of this private key as follows:
+
+         ```yaml title="aea-config.yaml"
+         (...)
+         all_participants: ${list:["0x15d34AAf54267DB7D7c367839AAf71A00a2C6A65"]}
+         ```
```

Comment:
Do we really require the all participants list for single agent run? And also it would be nice to include a concrete example with hello world agent

Reply:
RE 1. Yes. It is required. It launches an error otherwise.